modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -75,22 +94,41 @@ const Actions = ({fdsView, namespace, spritemap}: IFDSViewSectionProps) => {
 			});
 		}
 
-		if (activeSection === SECTIONS.EDIT_ITEM_ACTION) {
+		if (
+			activeSection === SECTIONS.EDIT_ITEM_ACTION ||
+			activeSection === SECTIONS.EDIT_CREATION_ACTION
+		) {
 			breadcrumbItems.push({
 				active: true,
 				label: initialActionFormValues?.label || '',
 			});
 		}
 
+		if (activeSection === SECTIONS.NEW_CREATION_ACTION) {
+			breadcrumbItems.push({
+				active: true,
+				label: Liferay.Language.get('new-creation-action'),
+				onClick: () => setActiveSection(SECTIONS.NEW_CREATION_ACTION),
+			});
+		}
+
 		return breadcrumbItems;
 	};
 
 	const loadFDSActions = async () => {
 		setLoading(true);
 
-		const response = await fetch(
-			`${API_URL.FDS_ACTIONS}?filter=(${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_ID} eq '${fdsView.id}')&nestedFields=${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION}&sort=dateCreated:desc`
-		);
+		let url = '';
+		if (activeTab === 0) {
+			setActiveSection(SECTIONS.ITEM_ACTIONS);
+			url = `${API_URL.FDS_ACTIONS}?filter=(${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_ITEM_ID} eq '${fdsView.id}')&nestedFields=${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_ITEM}&sort=dateCreated:desc`;
```

Comment:
We can simplify this, to avoid duplication. Maybe something like:
```js
const relationshipID = activeTab === 0 ? OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_ITEM_ID : OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_CREATION_ID:

const url = `${API_URL.FDS_ACTIONS}?filter=(${relationshipID} eq '${fdsView.id}')&...`;
```

I don't mind ternary expression here and elsewhere in PR. We do have mockups for a third tab, `Bulk Actions`, but   we will almost certainly end up implementing them as a part of item actions.

Reply:
:+1:  Just trying to keep tab related stuff together.